.github/workflows/windows-build.yml

```diff
@@ -0,0 +1,135 @@
+name: Build Windows Installer
+
+on:
+  push:
```

Comment:
```suggestion
  # TEMP: Enable PR trigger so this workflow runs in the PR that introduces it.
  # TODO_IN_THIS_PR(@bryanchriswhite): Remove before merging to avoid running on every PR.
  pull_request:
  
  push:
```